src/examples/wasm/export/wasm.go

```diff
@@ -8,12 +8,12 @@ import (
 func main() {
 }
 
-//export add
+//go:wasmimport add
```

Comment:
I think this update is premature, go:wasmimport is only supported on functions with no body that get linked at runtime to exports of other modules. 

Here it seems that the //export directive is intended to ask the compiler to add the function to the list of exports of the module, or am I missing something?

Reply:
Ah OK. I totally misunderstood what was happening here. Thanks for the clarification.